examples/CloudFunction/gsuite-exporter-cloudfunction/README.md

```diff
@@ -0,0 +1,81 @@
+# GSuite Exporter
+
+This package handles exporting data from the GSuite Admin APIs to a destination.
+
+The following GSuite Admin APIs are currently supported:
+
+- `reports_v1` - [Reports API](https://developers.google.com/admin-sdk/reports/v1/get-start/getting-started)
+    - `admin` - [Admin activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-admin.html)
+    - `drive` - [Google Drive activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-drive.html)
+    - `login` - [Login activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-login.html)
+    - `mobile` - [Mobile activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-mobile.html)
+    - `token` - [Authorization Token activity reports](https://developers.google.com/admin-sdk/reports/v1/guides/manage-audit-tokens.html)
+
+The following destinations are currently supported:
+- [Stackdriver Logging](https://cloud.google.com/logging/docs/)
+
+## Requirements
+* A GSuite Admin account
+* A service account with:
+  * [GSuite domain-wide delegation](https://developers.google.com/admin-sdk/reports/v1/guides/delegation) enabled.
+  * The IAM role `roles/iam.tokenCreator` set on the organization.
+
+### Collectors
```

Comment:
Remove from here till the end - this is already present in the GSuite Exporter documentation.

Reply:
Removed